Configs/en-ne_ec.yml

```diff
@@ -25,7 +25,7 @@ TransferH: false # switch to hard gumbel after the model reaches target accuracy
 seed: 1
 mode: emergent_communication
 language_model_lambda: 0.125
-language_model_path: Output/mbart_lm_lr6e-6
+language_model_path: DataLink/../Output/mbart_lm_lr6e-6
```

Comment:
Some places have `./DataLink` and some don't.  Is there a reason not to?  If not, maybe make all of these consistent with `./` (several places)

Reply:
No specific reason I had in mind. We can think about this some more but I'll make everything consistent for now!